zetaclient/testutils/stub/tss_signer.go

```diff
@@ -76,7 +78,16 @@ func (s *TSS) BTCAddress() string {
 }
 
 func (s *TSS) BTCAddressWitnessPubkeyHash() *btcutil.AddressWitnessPubKeyHash {
-	return nil
+	net, err := common.GetBTCChainParams(s.chain.ChainId)
+	if err != nil {
+		panic(err)
+	}
+	tssAddress := s.BTCAddress()
+	addr, err := btcutil.DecodeAddress(tssAddress, net)
+	if err != nil {
+		return nil
+	}
+	return addr.(*btcutil.AddressWitnessPubKeyHash)
```

Comment:
Would be better:
- add error returnin function
- return error instead of panic
- return error at l88
- return error if the addr can't be converted and of function

Reply:
I see this is for the testing.
I think we could provide `testing.T` to the function and use `require.NoError`

Reply:
The method `BTCAddressWitnessPubkeyHash()` is defined by `interfaces.TSSSigner` to which we cannot add argument of `testing.T`